test/util.js

```diff
@@ -19,14 +19,16 @@ export class MockLogger {
 }
 
 // Triggers an event that does not bubble.
-export const trigger = (eventName, wrapper) => {
-  wrapper.trigger(eventName);
-  return Vue.nextTick();
+export const trigger = (eventName, wrapper, selector = undefined) => {
+  const target = selector == null ? wrapper : wrapper.first(selector);
+  target.trigger(eventName);
+  const nextTick = Vue.nextTick();
+  return selector == null ? nextTick : nextTick.then(() => wrapper);
```

Comment:
i don't really understand this line. why return the wrapper only if you scoped the wrapper to selector?

Reply:
This is definitely confusing as-is... Ideally I'd like to return the wrapper all the time. The function hadn't previously been returning the wrapper, and I didn't have the chance to make sure that changing it wouldn't break any existing promise chain, so I kept the current behavior for the non-selector case and added the return-wrapper behavior for the new selector case. I've created an [issue](https://github.com/opendatakit/central-frontend/issues/133) to track this, so I should be able to make that change soon.